bitnami/influxdb/templates/influxdb/configmap-backup.yaml

```diff
@@ -43,7 +43,7 @@ data:
             echo "backuping ${DATABASE} db to {{ .Values.backup.directory }}/${DATABASE}"
             mkdir -p {{ .Values.backup.directory }}/${DATABASE}
 
-            influxd backup -host "${host}" -port {{ .Values.influxdb.service.port }} -portable -db ${DATABASE} {{ .Values.backup.directory }}/"${DATABASE}/${DATE}"
+            influxd backup -host "${host}:{{ .Values.influxdb.service.rpcPort }}" -portable -db ${DATABASE} {{ .Values.backup.directory }}/"${DATABASE}/${DATE}"
```

Comment:
Could you explain why this change is needed? Weren't the `service.port` the correct port to perform the backup?

Reply:
@miguelaeh Thanks for the comment! With the service port it simply doesn't work, as the remote connection is used here, and backups and restore is done by RPC port. I guess it hasn't been never tested before. The change is according the documentation here: 

https://docs.influxdata.com/influxdb/v1.8/administration/ports/#8088

https://docs.influxdata.com/influxdb/v1.8/administration/backup_and_restore/#online-backup-and-restore-for-influxdb-oss

> The default port used by the RPC service for RPC calls made by the CLI for backup and restore operations

Reply:
perfect! 👍